templates/desktop/index.html

```diff
@@ -364,7 +364,7 @@ <h3>Ubuntu Kylin</h3>
       </p>
     </div>
     <div class="col-6">
-      <img src="/static/img/homepage/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
+      <img src="http://www.ubuntukylin.com/public/images/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
```

Comment:
Why not use the asset server here?

Reply:
They updated the laptop picture and created a request in Issues #302, so I use that image link.